DR: Purpose Statement (#32)
blindnet-io/openness-framework: docs/decision-records/DR-0003-purpose-statement.md

# Defining our Collective Purpose Statement

| Status     | accepted                                                        |
| :--------- | :-------------------------------------------------------------- |
| **PR #**   | [32](https://github.com/blindnet-io/openness-framework/pull/32) |
| **Author** | Noël Macé (noel@blindnet.io)                                    |

## Context and Problem Statement

Before making this decision, blindnet only had a mission statement:

> « Helping developers make software that users love and trust. »

> A mission only factually state what an organization is **currently trying** to accomplish.

Reviewer: ```suggestion
A mission only factually states **what** an organization is trying to accomplish.
```
(HBR says "Mission" is about WHAT, and "Purpose is about "WHY")

> A mission only factually state what an organization is **currently trying** to accomplish.

Reviewer: ```suggestion
A mission only factually states **what** an organization is trying to accomplish.
```
(HBR says "Mission" is about WHAT, and "Purpose is about "WHY")

> A purpose statement stays necessary to simply state why the organization exist as a collective, and what drives this collective on a daily basis.

Reviewer: ```suggestion
A purpose statement stays necessary to simply state why the organization exist as a collective, and what drives this collective on a daily basis.
```
```suggestion
A purpose statement is necessary to  state **why** the organization exist as a collective, and what drives this collective on a daily basis.
```
HBR says "Mission" is about WHAT, and "Purpose is about "WHY"

> A purpose statement stays necessary to simply state why the organization exist as a collective, and what drives this collective on a daily basis.

Reviewer: ```suggestion
A purpose statement stays necessary to simply state why the organization exist as a collective, and what drives this collective on a daily basis.
```
```suggestion
A purpose statement is necessary to  state **why** the organization exist as a collective, and what drives this collective on a daily basis.
```
HBR says "Mission" is about WHAT, and "Purpose is about "WHY"

A [thorough collective discussion](https://github.com/blindnet-io/communication-management/issues/86) involving all blindnet's staff members has been conducted to address this matter, leading to a simple and straightforward purpose statement everyone agrees on and embrace.

## Decision Drivers

The purpose statement has to:

- state our common values and goals
- be straightforward
- avoid any confusing, ambiguous or jargonistic term
- be short (few words)
- be easy to remember and say

> 

Reviewer: ```suggestion

The purpose statement is not:
-  slogan
- a vision statement
```

> 

Reviewer: ```suggestion

The purpose statement is not:
-  slogan
- a vision statement
```

### Variations around "Empowering"

> see the definition of ["Power"](https://dictionary.cambridge.org/dictionary/english/power) in the Cambridge Dictionary

- Good, because Empowerment has a much greater and deeper impact than enablement. We empower people, organizations, products with trust.
- Bad, because "power" is ambiguous and might be linked in some contexts with authority, which is often confused with authoritarianism.
- Bad, because some might consider the lexical field of "power" is strongly linked with a "negative control" / coercion (over people, of something).

### Removing the word "Privacy"

- Good, because some might still consider "Privacy" jargonistic and redundant with "Trust".
- Bad, because most people still seems to have at least some notion of what it means.
- Bad, because "Privacy" seems mandatory to allow our purpose statement to sound meaningful enough to developers.
- Bad, because "Trust" alone doesn't state on which condition the trust is built.

## Links

- [Notions of Privacy](https://github.com/blindnet-io/product-management/blob/main/refs/notion-of-privacy/notion-of-privacy.md)

> 

Reviewer: ```suggestion
- [HBR: The Difference Between Purpose and Mission ](https://hbr.org/sponsored/2016/02/the-difference-between-purpose-and-mission)
```

> 

Reviewer: ```suggestion
- [HBR: The Difference Between Purpose and Mission ](https://hbr.org/sponsored/2016/02/the-difference-between-purpose-and-mission)
```
